server/controllers/discover.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Controller.
+ */
+module.exports = function(app) {
+
+  const models = app.set('models');
+
+  return (req, res, next) => {
+    const show = req.query.show;
+    const sort = req.query.sort;
+    const offset = req.query.offset;
+    models.Group.getGroupsSummaryByTag(
+      !show || show === 'all' ? '' : show, 
+      12,
+      [], 
+      100, 
+      false,
+      'g."createdAt"',
+      sort === 'newest' ? 'desc' : 'asc',
```

Comment:
You are right, directly passing `sort` will introduce a SQL injection vulnerability since it goes straight to a raw query. So, in order to pass sort directly, we must **escape** it before passing it to the `queries.getGroupsByTag` method.

So it can be either a ternary condition or a function call to the *escape* function